test/unit/pad-utils.test.ts

```diff
@@ -0,0 +1,20 @@
+import { JSDOM } from 'jsdom'
+import * as RdfLib from 'rdflib'
+import { getChunks, notepadToHTML, xmlEncode } from '../../src/pad-utils'
+
+describe('getChunks', () => {
+  it('exists', () => {
+    expect(getChunks).toBeInstanceOf(Function)
+  })
+})
+
+describe('notepadToHTML', () => {
+  it('exists', () => {
```

Comment:
Please lets stop writing these tests - TypeScript does this for us. Write tests that make sure that the functions are exported correctly to the API of Solid UI, e.g. check out what I did for [tabWidget in the tabs module](https://github.com/solid/solid-ui/pull/247/files#diff-2d70df5dd69be0ab433b268762ec8b97R27).

Reply:
I see what you've done here...  on this one in particular getChunks it is not exposed on the public api.  I guess I should just check that it runs and the functionality within?

Reply:
Yeah, if it's an internal method, you want to check that it does what you intend, no need to check the public API.